### ISSUE TITLE
ocp3 shown offline is rhacm, despite being online

### ISSUE THREAD

**Issue by DanielFroehlich** [stormshift/support#48]
OCP3 Cluster is shown offline in RHACM
![image](https://user-images.githubusercontent.com/18532443/139215801-0a7b3816-8b56-434f-a17b-ed5287c54bf9.png)

But the cluster is alive and kicking.

**Comment by manfredmuth** [comment]
Seems there are still weird problems with ocp3
For the last days ( no clue when it started ) login to ocp3 is not working.
Local admin login does not work  -> you are straight redirected to the login page without other notice
SSO login  -> you are straight redirected to the login page without other notice

**Comment by DanielFroehlich** [comment]
ocp3 issues fixed in dec, not sure why rhacm did not pick it up again. I suspect some expired keys or certs, as the cluster was offline for quite some while.
I fixed the issue by detachting the cluster in RHACM, cleaning the ocp3 cluster (by executing rhacm/cleanup_rhacm_agent.sh from the automation scripts).
Then, I did re-import the ocp3 cluster by running 
`ansible-playbook -i cfg/inventory.yml -e @cfg/stormshift.yml -e @cfg/ocp3.yml 420_rhacm_importManagedCluster.yml
`
Now it is back visible in RHACM
